notebooks/ch10/modeling_abs_huber.md

```diff
@@ -317,57 +324,66 @@ def points_and_cost(y_vals, xlim, cost_fn=abs_cost):
     plt.xlabel('Points')
     
     ax = plt.subplot(122)
-    plt.plot(thetas, abs_costs)
+    plt.plot(thetas, abs_losses)
     plt.xlim(*xlim)
     plt.xlabel(r'$ \theta $')
-    plt.ylabel('Cost')
+    plt.ylabel('Loss')
     plt.legend()
```

Comment:
Looks like we should remove the `plt.legend()` line since there's nothing to put in the legend. This will also get rid of the warnings below.